src/Welcome.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import WelcomeImage from './img/Octocat.jpg';
+import WelcomeLogo from './img/WelcomeLogo.png';
+import './Welcome.css';
+
+const CLIENT_ID = 'c6b85184ff709b73b677'
+const REDIRECT_URI = 'http://localhost:3000/Home'
+
+class Welcome extends React.Component {
+    render() {
+        return (
+            <div>
+                <nav className="navbar navbar-inverse">
+                    <div className="container-fluid">
+                        <div className="navbar-header">
+                            <a href="#" className="navbar-left"><img src={WelcomeLogo} height="50" alt="Welcome Logo" /></a>
+                            <a className="navbar-brand" href="#">
+                                <span id="dashboard">GitHub-Dashboard</span>
+                            </a>
+                        </div>
+                    </div>
+                </nav>
+
+                <div className="container">
+                    <div className="row">
+                        <a href="#" className="navbar-left"><img src={WelcomeImage} height="400" alt="Welcome" /></a>
```

Comment:
I think the link should be removed from this image, so that line 26 is just:

`<img src={WelcomeImage} height="380" alt="Welcome" />`

Reply:
Working on it!

Reply:
Ayyyy, alright, missed that, makes sense